tests/test_coreg/test_base.py

```diff
@@ -960,8 +960,7 @@ def test_blockwise_coreg_large_gaps(self) -> None:
         ddem_post = (aligned - self.ref).data.compressed()
         ddem_pre = (tba - self.ref).data.compressed()
         assert abs(np.nanmedian(ddem_pre)) > abs(np.nanmedian(ddem_post))
-        # TODO: Figure out why STD here is larger since PR #530
-        # assert np.nanstd(ddem_pre) > np.nanstd(ddem_post)
```

Comment:
Why is an identical assertion commented out here?

Reply:
This is linked to #584